src/NzbDrone.Core.Test/IndexerTests/GetstrikeTests/GetstrikeFixture.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Moq;
+using NUnit.Framework;
+using NzbDrone.Common.Http;
+using NzbDrone.Core.Indexers;
+using NzbDrone.Core.Indexers.GetStrike;
+using NzbDrone.Core.Parser.Model;
+using NzbDrone.Core.Test.Framework;
+using NzbDrone.Test.Common;
+using System;
+using System.Linq;
+using FluentAssertions;
+using NzbDrone.Core.IndexerSearch.Definitions;
+using System.Collections.Generic;
+
+namespace NzbDrone.Core.Test.IndexerTests.GetstrikeTests
+{
+    [TestFixture]
+    public class getStrikeFixture : CoreTest<GetStrike>
+    {
+        [SetUp]
+        public void Setup()
+        {
+            Subject.Definition = new IndexerDefinition
+                {
+                    Name = "Getstrike",
+                    Settings = new GetStrikeSettings(),
+                };
+        }
+
+        [Test]
+        public void should_parse_top_feed_from_Getstrike()
+        {
+            var recentFeed = ReadAllText(@"Files/Indexers/Getstrike/Getstrike.json");
+
+            Mocker.GetMock<IHttpClient>()
+                .Setup(o => o.Execute(It.Is<HttpRequest>(v => v.Method == HttpMethod.GET)))
+                .Returns<HttpRequest>(r => new HttpResponse(r, new HttpHeader(), recentFeed));
+
+            SingleEpisodeSearchCriteria s = new SingleEpisodeSearchCriteria();
+            SingleEpisodeSearchCriteria searchCriteria = new SingleEpisodeSearchCriteria
```

Comment:
please use `var searchCriteria` instead of `SingleEpisodeSearchCriteria searchCriteria `